scripts/k8s/create-secret.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Check if the COMPANION_CONFIG_BASE64 env is set.
+if [ -z "$COMPANION_CONFIG_BASE64" ]; then
+  echo "Error: COMPANION_CONFIG_BASE64 is not set."
+fi
+
+echo "Creating secret companion-config in ai-system namespace."
+# create a secret with the base64 encoded file.
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: companion-config
+  namespace: ai-system
```

Comment:
nitpick: maybe set the `name` and `namespace` with variables as the secret is used in the resources and jobs.

Reply:
the name and namespace is also hardcoded in the manifests. So making it variable would not help much.

Reply:
you can take care of this one in the follow-up PR